From 587fbb62dfc59ef28c21224427daaccd0ad3ee58 Mon Sep 17 00:00:00 2001
From: Zheng Zengkai <zhengzengkai@huawei.com>
Date: Tue, 27 Jul 2021 15:10:39 +0800
Subject: [PATCH] openeuler_defconfig: Enable CONFIG_SMMU_BYPASS_DEV by default
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 587fbb62dfc59ef28c21224427daaccd0ad3ee58


euleros inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I42DAP
CVE: NA

----------------------------------

Some manufacturers want to enable SMMU for using virtualization
pass-through feature on the server with the 3408iMR/3416iMR RAID
card installed.

Enable CONFIG_SMMU_BYPASS_DEV by default in openeuler_defconfig.

Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 248ceeefec7f..cb7ea0f4b282 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -5803,6 +5803,7 @@ CONFIG_ARM_SMMU=y
 CONFIG_ARM_SMMU_DISABLE_BYPASS_BY_DEFAULT=y
 CONFIG_ARM_SMMU_V3=y
 # CONFIG_ARM_SMMU_V3_SVA is not set
+CONFIG_SMMU_BYPASS_DEV=y
 # CONFIG_QCOM_IOMMU is not set
 # CONFIG_VIRTIO_IOMMU is not set
 
-- 
2.26.2

